java/src/main/java/ai/rapids/cudf/Aggregation.java

```diff
@@ -1,6 +1,6 @@
 /*
  *
- *  Copyright (c) 2020-2023, NVIDIA CORPORATION.
+ *  Copyright (c) 2020-2025, NVIDIA CORPORATION.
```

Comment:
Change the copyright header for the files that were forgotten in the previous PR. Github CI seems does not check on Java files.